distributions/demo-server/build.gradle

```diff
@@ -184,4 +184,15 @@ task installToolsAndDockerize {
     dependsOn ":client-rest:installTools", dockerize
 }
 
+configurations.all {
```

Comment:
Is this related to this PR?

Reply:
Only tangentially, you can't run DTP right now with out this change, so I included it in this PR